com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollServiceInProgressOperationsExecution.java

```diff
@@ -75,8 +82,17 @@ private boolean isServiceDeleted(ExecutionWrapper execution, UUID uuid) {
     }
 
     @SuppressWarnings("unchecked")
-    private ServiceOperation getLastOperation(Map<String, Object> cloudServiceInstance) {
+    protected ServiceOperation getLastOperation(Map<String, Object> cloudServiceInstance) {
         Map<String, Object> lastOperationAsMap = (Map<String, Object>) cloudServiceInstance.get(ServiceOperation.LAST_SERVICE_OPERATION);
-        return ServiceOperation.fromMap(lastOperationAsMap);
+        ServiceOperation lastOperation = ServiceOperation.fromMap(lastOperationAsMap);
+        return handleFailedOperationWithoutDescription(lastOperation);
+    }
+
+    private ServiceOperation handleFailedOperationWithoutDescription(ServiceOperation lastOperation) {
```

Comment:
This method doesn't check operation type (create,update or delete). If the operation is in failed state and doesn't have description it will add default message for description.

Reply:
I can see that, but i didn't thought of better place to ask my question.